qa/suites/upgrade/jewel-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -36,9 +36,3 @@ upgrade-sequence:
        daemons: [osd.2, osd.3]
        wait-for-healthy: false
        wait-for-osds-up: true
-   - exec:
```

Comment:
@smithfarm i am not sure why we want to backport 468285b65d6bef2d86c80ebdfecf0920294ca5cd in two separated commits: 

- edd504c593bceac2059f475621177951c70a47e4
- a17afe0e1e50520996ff46bb0e81955fca11b823